docs/UserGuide.md

```diff
@@ -794,19 +914,16 @@ This feature **organises clients** based on **how soon their next birthday** wil
 
 <box type="info" header="Notes">
 
-* The current date is based on the local system's time.
-* The calculation is based on the number of days until their next upcoming birthday.
+* The current date is based on the local system's time. 
+* If their birthday has already passed, the calculation is based on the number of days until their next birthday next year.
 </box>
 
-[//]: # ()
-[//]: # (<box type="warning" header="Warning">)
-
-[//]: # ()
-[//]: # (* If the list presented is currently a filtered list after using [filter]&#40;#filtering-clients-filter&#41;,)
 
-[//]: # (  sort will work on the new filtered list.)
+<box type="warning" header="Warning">
 
-[//]: # (</box>)
+* If the list presented is currently a filtered list after using [filter](#filtering-clients-filter), sort will work on the new filtered list.
+* If a birthday falls on February 29th (leap day), the day calculation is based on March 1st if the year does not have a leap date as realistically, most would still celebrate every year.
+</box>
```

Comment:
I think can move these warnings to notes as they are not exactly cautionary, small thing LOL

Reply:
🌮